### ISSUE TITLE
 Incorrect work from to

### ISSUE THREAD

**Issue by Xurma2** [Hacker0x01/react-datepicker#281]
![image](https://cloud.githubusercontent.com/assets/5157416/11965291/cc7c965e-a904-11e5-9ac3-0b4f51845776.png)

Specifically does not work you can see - image. 
Every new click on the input there is a shift in the date of the month.
What can be wrong?

I used the code from your example.
My code:

```	
getInitialState() {
		return {
			startDate: moment(),
			endDate: moment()
		}
	},

	handleChange(ref) {
		let startDate = ref.startDate,
			  endDate = ref.endDate;

		if (startDate.isAfter(endDate)) {
			let temp = startDate;
			startDate = endDate;
			endDate = temp;
		}

		this.setState({
			startDate: startDate,
			endDate: endDate
		});
	},

	handleChangeStart(startDate) {
		this.handleChange({ startDate: startDate });
	},

	handleChangeEnd(endDate) {
		this.handleChange({ endDate: endDate });
	},



								<DatePicker
								            selected={this.state.startDate}
														startDate={this.state.startDate}
								            endDate={this.state.endDate}
														onChange={this.handleChangeStart}
								/>

								<DatePicker

								            selected={this.state.endDate}
														startDate={this.state.startDate}
								            endDate={this.state.endDate}
														onChange={this.handleChangeEnd}
								/>

```

**Comment by brandonfancher** [comment]
I'm having this same exact problem.

**Comment by rafeememon** [comment]
This might be fixed with v0.18.0, can you try it with that version?

Re: #287

**Comment by TrofimenkoAI** [comment]
Hello guys, I really enjoy date picker, but we have a little bug, when date fields jumps up and down. I mean when select date current position upper on input. Picker change his position randomly when you select month in different ways :(
(Sorry for bad English)
